### ISSUE TITLE
Split Tunneling block all connections

### ISSUE THREAD

**Issue by berhram** [amnezia-vpn/amnezia-client#510]
**Describe the bug**
While split tunneling is on, none of the sites (whether in the allowed list or not) will open.

**To Reproduce**
1. Turn split tunneling.
2. Click on connect.
3. Done

**Expected behavior**
Allowed blocked sites will open

**Log files**
[AmneziaVPN.log](https://github.com/amnezia-vpn/amnezia-client/files/13985671/AmneziaVPN.log)

**Desktop (please complete the following information):**
 - OS: Windows 11
 - Version 4.2.0.1 (and any previos)

**Additional context**
Using VPN config provided by VPN Generator

**Comment by helgiftw** [comment]
same issue on android client

**Comment by pokamest** [comment]
@berhram Could you also provide service log.
C:\ProgramData\AmneziaVPN\log

@helgiftw could you provide app log

**Comment by berhram** [comment]
> @berhram Could you also provide service log. C:\ProgramData\AmneziaVPN\log
> 
> @helgiftw could you provide app log

[AmneziaVPN-service.log](https://github.com/amnezia-vpn/amnezia-client/files/14171833/AmneziaVPN-service.log)

**Comment by Korb** [comment]
> could you provide app log

@pokamest, where in case of Google Android 14 can I find the Amnezia VPN 4.8.1.9 log file? Of course, after enabling it in the application settings.

**Comment by WofWca** [comment]
Could it have to do with when the server was set up, or the server (as opposed to client) setup in general?
I experience the same issue with some of my servers, on Windows on 4.8.2.3.
I tried split tunneling with different protocols (WireGuard, OpenVPN, OpenVPN over Cloak) on one server and it never worked for that server.
But it does work alright on two other servers, one of which I set up recently with OpenVPN and OpenVPN over Cloak.

To summarize, this could be a server issue?

**Comment by Nethius** [comment]
Thank you for your report. Please contact our support team via chat at https://t.me/amnezia_support_bot or via email at [support@amnezia.org](mailto:support@amnezia.org). It would be helpful if you could include a link to this issue in your message.

Our team will review the details and assist you in resolving the problem.
